test/unit/pdf_find_controller_spec.js

```diff
@@ -264,5 +265,327 @@ describe("pdf_find_controller", function () {
       pageMatches: [[19, 46, 62]],
       pageMatchesLength: [[8, 8, 8]],
     });
+
+    await testSearch({
+      eventBus,
+      pdfFindController,
+      state: {
+        query: "1/2",
+        caseSensitive: false,
+        entireWord: false,
+        phraseSearch: true,
+        findPrevious: false,
```

Comment:
Nit: Here, and elsewhere below, note that you now only need to specify those options that *differ* from the default values in the `testSearch` function.
The advantage of using a more compact format in the individual test-cases is that it becomes easier to spot what's "special" about each test-case.